lib/knapsack_pro/runners/queue/rspec_runner.rb

```diff
@@ -21,6 +21,7 @@ def self.run(args, stream_error = $stderr, stream_out = $stdout)
         end
 
         def initialize(adapter_class, rspec_pure, args, stream_error, stream_out)
+          ENV['KNAPSACK_PRO_RSPEC_OPTIONS'] = args
```

Comment:
Have you considered doing it (and for Regular) in the Rake task?

I don't have a strong opinion either way, so if this is simpler (eg, tests), I'm fine with it.

Reply:
Sometimes we are calling it args, sometimes options. I think we should stick with options for this one (vs positional arguments.)

Reply:
> Have you considered doing it (and for Regular) in the Rake task?

No.

> I don't have a strong opinion either way, so if this is simpler (eg, tests), I'm fine with it.

It's simpler to have it in runner classes like `lib/knapsack_pro/runners/queue/rspec_runner.rb` for testing purposes.

Also, we need to support the binary version `bin/knapsack_pro`, which directly calls runner classes.

Reply:
> Sometimes we are calling it args, sometimes options. I think we should stick with options for this one (vs positional arguments.)

Hmm. I wanted to use a general term.
Perhaps the option name could be used once we know these are actual options. I'm not sure if we need to be that specific right away.

We use the args term in many places, also in [the docs](https://docs.knapsackpro.com/ruby/reference/#command-line-arguments).

![image](https://github.com/user-attachments/assets/e510a8e1-6150-4d10-8471-11e5ec24dfd7)